perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark.py

```diff
@@ -339,20 +352,48 @@ def _parse_arguments(argv):
   dir_structure_present = _check_if_dir_structure_exists(dir_str)
   if not dir_structure_present:
     log.error("Test data does not exist.To create test data, run : \
-    python3 generate_folders_and_files.py <dir_config.json> ")
+        python3 generate_folders_and_files.py {} ".format(dir_config))
     sys.exit(1)
 
-  results = dict()  # Dict object to store the results corresonding to the test types.
-  _perform_testing(dir_str, "flat", args.num_samples, results)
-  flat_parsed_metrics = _parse_results(dir_str, results['flat'], args.num_samples)
-  upload_values_flat = _get_values_to_export(dir_str, flat_parsed_metrics,
-                                             "flat")
+  _perform_testing(dir_str, test_type, num_samples, results)
+  parsed_metrics = _parse_results(dir_str, results[test_type], num_samples)
+  upload_values = _get_values_to_export(dir_str, parsed_metrics,
+                                             test_type)
 
-  if args.upload_gs:
+  if upload_gs:
     log.info('Uploading files to the Google Sheet\n')
-    exit_code = _upload_to_gsheet(WORKSHEET_NAME_FLAT, upload_values_flat,
+    if test_type == "flat":
+      worksheet= WORKSHEET_NAME_FLAT
+    else:
+      worksheet= WORKSHEET_NAME_GCS
+
+    exit_code = _upload_to_gsheet(worksheet, upload_values,
                                   SPREADSHEET_ID)
     if exit_code != 0:
       log.error("Upload to gsheet failed!")
   else:
-    print(upload_values_flat)
+    print(upload_values)
+
+
+if __name__ == '__main__':
+  argv = sys.argv
+  if len(argv) < 2:
+    raise TypeError('Incorrect number of arguments.\n'
+                    'Usage: '
+                    'python3 renaming_benchmark.py  [--upload_gs] [--num_samples NUM_SAMPLES] config_file ')
```

Comment:
Let's pass `--bucket-type` as an argument and pass only one config-file at a time. This way, we can avoid the "if" conditions here.

Reply:
Updated.